modules/Sources/Features/BalanceBreakdown/BalanceBreakdownStore.swift

```diff
@@ -47,6 +47,10 @@ public struct BalanceBreakdownReducer: Reducer {
             isShieldingFunds || !isShieldableBalanceAvailable
         }
         
+        public var isProcessingZeroAvailableBalance: Bool {
+            totalBalance.amount != shieldedBalance.amount && shieldedBalance.amount == 0
```

Comment:
It could be the case that there is actually zero shielded balance (and non-zero transparent balance) when fully scanned, which would cause the spinner to show incorrectly. That can be resolved in #1087.